homeworks/hw8/hw8-2.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function someFunction(...names) {
+    console.log(`Greetings ${names.join(', ')}`)
+}
+
+function slower(func, seconds) {
+    return function (...names) {
```

Comment:
```suggestion
    return function (...names) {
```

А якщо ми задіємо наш декоратор для функцій аргументами яких не є names?)

Треба старатись писати максимально можливо абстрактні рішення. Рішення в цілому вірне, це не помилка, просто звертаю увагу на неймінг.